public/app/plugins/datasource/graphite/graphite_query.ts

```diff
@@ -326,3 +330,44 @@ export default class GraphiteQuery {
 function renderTagString(tag: { key: any; operator?: any; value?: any }) {
   return tag.key + tag.operator + tag.value;
 }
+
+/**
+ * mutates the second seriesByTag function into a string to fix a parsing bug
+ * @param astNode
+ * @param innerFunc
+ */
+// eslint-disable-next-line
+function handlemultipleSeriesByTagsParams(astNode: any, innerFunc: any) {
+  // if function is asPercent and has two params that are function seriesByTags keep the second as a string otherwise we have a parsing error
+  if (innerFunc.def.name === 'asPercent' && astNode.params.length >= 2) {
```

Comment:
so I was able to repro this issue with other functions besides asPercent, like consolidateBy. I know very little about graphite functions, so if you can take a look and make sure there aren't any other functions that take multiple param that could get formatted weird like this that would make me feel better.

Tl;DR https://graphite.readthedocs.io/en/latest/functions.html#graphite.render.functions.aliasByTags, could this impact all graphite functions that take a `seriesList`?

Reply:
Scratch that, I'm going to have to check all 105 functions for this 😦